src/models/thread.ts

```diff
@@ -38,9 +38,9 @@ export enum ThreadEvent {
     Delete = "Thread.delete",
 }
 
-type EmittedEvents = Exclude<ThreadEvent, ThreadEvent.New> | RoomEvent.Timeline | RoomEvent.TimelineReset;
+export type ThreadEmittedEvents = Exclude<ThreadEvent, ThreadEvent.New> | RoomEvent.Timeline | RoomEvent.TimelineReset;
 
-export type EventHandlerMap = {
+export type ThreadEventHandlerMap = {
```

Comment:
Changing this is a breaking change

Reply:
As I see it was never exported before, so it is just an internal type rename, or am I missing something?

Reply:
It was exported, you could import it as `matrix-js-sdk/src/models/thread` - you just changed its name, if you prefer the new name, fine, but you must maintain the old name with a deprecation notice